client/app/auth/service.js

```diff
@@ -0,0 +1,22 @@
+angular.module('throwcast.auth', [])
+
+.factory('authInterceptor', function ($q, $window){
+  return {
+    request: function (config) {
+      config.headers = config.headers || {};
+      if ($window.sessionStorage.token) {
+        config.headers.Authorization = 'Bearer ' + $window.sessionStorage.token;
+      }
+      return config;
+    },
+    response: function (rejection) {
+      if (response.status === 401) {
```

Comment:
I think we redirect to login on a rejection with response 401. Take a look online on how to handle that status code.